tests/WatcherTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the React Symfony Server package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ */
+
+declare(strict_types=1);
+
+namespace Drift\Server\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+/**
+ * Class WatcherTest.
+ */
+class WatcherTest extends TestCase
+{
+    /**
+     * Test default adapter static folder.
```

Comment:
Do we really test static folder here?) As I see it, we test that the `watch` command really works and at least starts the server.

Reply:
Not really. I do not really know how to test this feature. I will add an issue to test this.